lib/spectacles/schema_dumper.rb

```diff
@@ -8,11 +8,12 @@ def self.dump_views(stream, connection)
       end
     end
 
-    def self.dump_materialized_views(stream, connection)
+    def self.dump_materialized_views(dumper, stream, connection)
       unless (Spectacles.config.enable_schema_dump == false)
         if connection.supports_materialized_views?
           connection.materialized_views.sort.each do |view|
             dump_materialized_view(stream, connection, view)
+            dumper.send(:indexes, view, stream)
```

Comment:
I know it's not kosher to call private methods because they're susceptible to breaking and I cringed when I did this but I didn't want to repeat all the existing code I needed to accomplish this. Do you have any suggestions?